src/VisualStudio/Core/Def/PickMembers/PickMembersDialog.xaml

```diff
@@ -56,7 +63,8 @@
         <GroupBox x:Uid="MemberSelectionGroupBox"
                   Margin="0, 9, 0, 0"
                   Grid.Row="1"
-                  Header="{Binding ElementName=dialog, Path=PickMembersTitle}">
+                  Header="{Binding ElementName=dialog, Path=PickMembersTitle}"
+                  Foreground="{DynamicResource {x:Static vs:ThemedDialogColors.HeaderTextBrushKey}}">
```

Comment:
Is there a Theme color we should be using for the box Border?

Reply:
It seems to be automatically done with `vs:ThemedDialogStyleLoader.UseDefaultThemedDialogStyles="True"`